backend/src/main/java/capstoneexpensetracker/backend/Travelercontroller.java

```diff
@@ -0,0 +1,19 @@
+package capstoneexpensetracker.backend;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/traveler")
+@RequiredArgsConstructor
+public class Travelercontroller {
+
+    private final TravelerService travelerService;
+
+    @GetMapping
+    List<Traveler> displayConviatoresList() {
```

Comment:
Das Wort `Conviatores` kenne ich bislang nicht. Es ist aber durchaus in Ordnung, wenn Du für eine BusinessEntity/Domäne in Deiner Anwendung einen neuen Fachbegriff einführst.
Ist Traveler und Conviatores etwas unterschiedliches?

Reply:
conviatores ist eine alternative für mitreisende. Ich kann aber auch ein anderes Wort nutzen :D

Reply:
Nein, nein, lass ruhig. Da habe ich wieder einmal etwas gelernt 😊